File: engine/engine-model/src/main/scala/net/server/channel/packet/cash/operation/FriendshipRingPacket.scala
package net.server.channel.packet.cash.operation

class FriendshipRingPacket(private var _action: Int, private var _birthday: Int, private var _payment: Int, private var _sn: Int, private var _sentTo: String, private var _text: String) extends BaseCashOperationPacket(_action = _action) {
  def birthday: Int = _birthday

  def payment: Int = _payment

  def sn: Int = _sn

  def sentTo: String = _sentTo

  def text: String = _text
}

File: engine/engine-model/src/main/java/net/server/channel/packet/cash/operation/FriendshipRingPacket.java
package net.server.channel.packet.cash.operation;

public class FriendshipRingPacket extends BaseCashOperationPacket {
   private final Integer birthday;

   private final Integer payment;

   private final Integer sn;

   private final String sentTo;

   private final String text;

   public FriendshipRingPacket(Integer action, Integer birthday, Integer payment, Integer sn, String sentTo, String text) {
      super(action);
      this.birthday = birthday;
      this.payment = payment;
      this.sn = sn;
      this.sentTo = sentTo;
      this.text = text;
   }

   public Integer birthday() {
      return birthday;
   }

   public Integer payment() {
      return payment;
   }

   public Integer sn() {
      return sn;
   }

   public String sentTo() {
      return sentTo;
   }

   public String text() {
      return text;
   }
}
